lib/src/main/scala/spinal/lib/bus/avalon/AvalonMMSlaveFactory.scala

```diff
@@ -7,12 +7,13 @@ import scala.collection.Seq
 
 object AvalonMMSlaveFactory{
   def getAvalonConfig(addressWidth: Int,
-                      dataWidth: Int) = {
+                      dataWidth: Int,
+                      useByteEnable: Boolean) = {
```

Comment:
I think this should have default value false, to be backward compatible

Reply:
fixed with additional commit.